x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authc/TokenAuthIntegTests.java

```diff
@@ -740,37 +739,45 @@ public void testRefreshAsDifferentUser() throws IOException {
         assertThat(e.getCause().getMessage(), containsString("tokens must be refreshed by the creating client"));
     }
 
-    public void testCreateThenRefreshAsDifferentUser() throws IOException {
-        final RequestOptions superuserOptions = RequestOptions.DEFAULT.toBuilder()
+    public void testCreateThenRefreshAsRunAsUser() throws IOException {
```

Comment:
The orignal test was almost identical to `testRefreshingToken`. I therefore recycled it into a token create/refresh with run-as .